target/linux/at91/image/legacy.mk

```diff
@@ -0,0 +1,97 @@
+define Device/at91sam9263ek
+  DEVICE_TITLE := Atmel AT91SAM9263-EK
+endef
+TARGET_DEVICES += at91sam9263ek
+
+define Device/at91sam9g15ek
+  DEVICE_TITLE := Atmel AT91SAM9G15-EK
+endef
+TARGET_DEVICES += at91sam9g15ek
+
+define Device/at91sam9g20ek
+  DEVICE_TITLE := Atmel AT91SAM9G20-EK
+  KERNEL := kernel-bin | lzma | uImage lzma
+  IMAGE/root.ubi := append-ubi | install-dtb
```

Comment:
This is misleading. The install-dtb build step isn't related/doesn't change the created image. Please do it this way:

```
define Device/at91sam9g20ek
  DEVICE_TITLE := Atmel AT91SAM9G20-EK
  KERNEL := kernel-bin | lzma | uImage lzma
  IMAGES += dtb
  IMAGE/dtb := install-dtb
define Device/at91sam9g20ek
```

Please do it the same way for all other images which need the dtb file.